tf-modules/credstash-setup/variables.tf

```diff
@@ -27,3 +31,14 @@ variable "db_table_name" {
   default = "credential-store"
   description = "Name of the DynamoDB table where credentials will be stored"
 }
+
+variable "create_reader_policy" {
+  default = false
+  description = "Should credstash Secret Reader IAM Policy be created."
+}
+
+variable "create_writer_policy" {
+  default = false
+  description = "Should credstash Secret Writer IAM Policy be created."
+}
+
```

Comment:
A: Those are minimal policies to be able to read/write from DynamoDB table. Use can create less restrictive policies and attach to the same role created by the user if needed, all information necessary for that is exported by the module.
A: Many policies can be attached to a single role, that's why it is not a role that is created in this module, but the policies, which can be used later if needed. So, no, nothing here would need to be changed if a user wants to access EBS volume S3, what have you, he can create another policy and attach to the same role.